content/en/hosting/monitoring/setup.md

```diff
@@ -141,19 +141,36 @@ With the [release of 1.1.0](https://github.com/medic/cht-watchdog/releases/tag/1
 
 {{% alert title="Note" %}}
 Always run this longer version of the `docker compose` command which specifies both compose files for all future [upgrades](#upgrading).
+This configuration also works for [couch2pg]({{< relref "apps/tutorials/couch2pg-setup" >}} data. Edit `sql_servers.yml` to set the value of `collectors` to `couch2pg`.
```

Comment:
I think we should weave in the CHT Sync & couch2pg compatibility elsewhere and not conflate it with the `docker compose` run  

```suggestion
```